Content.Server/Nutrition/Components/FoodComponent.cs

```diff
@@ -37,21 +22,21 @@ public class FoodComponent : Component, IUse, IAfterInteract
 
         [ViewVariables]
         [DataField("useSound")]
-        private SoundSpecifier UseSound { get; set; } = new SoundPathSpecifier("/Audio/Items/eatfood.ogg");
+        public SoundSpecifier UseSound { get; set; } = new SoundPathSpecifier("/Audio/Items/eatfood.ogg");
 
         [ViewVariables]
         [DataField("trash", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
-        private string? TrashPrototype { get; set; }
+        public string? TrashPrototype { get; set; }
 
         [ViewVariables]
         [DataField("transferAmount")]
-        private FixedPoint2? TransferAmount { get; set; } = FixedPoint2.New(5);
+        public FixedPoint2? TransferAmount { get; set; } = FixedPoint2.New(5);
 
-        [DataField("utensilsNeeded")]
-        private UtensilType _utensilsNeeded = UtensilType.None;
+        [DataField("optionalUtensil")]
+        public UtensilType OptionalUtensil = UtensilType.Fork; //There are more "solid" than "liquid" food
```

Comment:
doesn't this get rid of the 'required utensil' mechanic? some stuff in SS13 does use it and it'd be nice to keep it as a possibility

Reply:
okay, I can make bool field "required" and check for this, is that gonna be okay?

Reply:
Actually no! I've searched on TG and only thing that uses fork is "cheese omelette" but you can eat it even without fork...
On Paradise you can eat everything with or without fork, even soups... Just imagine what it's like to die of hunger because chef made bunch of food uneatable without utensils which was stolen by someone.
Maybe I could make it like on TG or Paradise where the fork or spoon is actually a container and which shows what it contains with tinted texture, but that for another PR.
Now it works like you can eat it with utensil or you can eat it with just your mouth, making it utensil-only would be awful I guess

Reply:
Just checked VG, same thing 
![image](https://user-images.githubusercontent.com/36813380/142583026-c6a51b50-a77c-490d-91e6-33b2c3da62ed.png)
It just checks "is utensil valid for this food"

Reply:
So, now there are two fields, one is "Utensil" which determines appropriate utensil(-s) for this food, second one is "Required" field which tells you "you need to hold this utensil to eat that", but I'm assure you that no one gonna use this field

Reply:
that's fine then